plugins/techdocs-backend/examples/documented-component/docs/code/code-sample.md

```diff
@@ -20,3 +20,12 @@ const DefaultEntityPage = ({ entity }: { entity: Entity }) => (
   </EntityPageLayout>
 );
 ```
+
+Here is an example of Python code:
+
+```python
+def getUsersInGroup( targetGroup, secure = False ):
+
+    if __debug__:
+        print ('targetGroup=[' + targetGroup + ']')
```

Comment:
```suggestion
def getUsersInGroup(targetGroup, secure=False):

    if __debug__:
        print('targetGroup=[' + targetGroup + ']')
```

[PEP8 speaks](https://github.com/orkohunter/pep8speaks) in my head :D

Reply:
I'm still old school two spaces after a period kinda guy, it's so hard not to hit the keyboard ha ha